roles/lib_utils/src/generate.py

```diff
@@ -27,19 +27,47 @@ def parse_args():
     return parser.parse_args()
 
 
+def fragment_banner(fragment_path, side, data):
```

Comment:
Could this and the above library be combined or is it required to have two copies due to ansible restrictions?